x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -522,6 +522,18 @@ export class TaskRunner<
         namespace,
       },
     };
+
+    const startEvent = cloneDeep({
+      ...event,
+      event: {
+        ...event.event,
+        action: EVENT_LOG_ACTIONS.executeStart,
+        start: new Date().toISOString(),
```

Comment:
`eventLogger.startTiming(event)` adds a `start` time to the event so maybe you could clone after calling `eventLogger.startTiming`? Otherwise their start times will be just slightly different which may be confusing.

Reply:
Good tip! Updated now